x-pack/filebeat/module/suricata/eve/ingest/pipeline.json

```diff
@@ -54,6 +54,13 @@
       ,"ignore_missing": true
       }
     }
+  , {"convert":
```

Comment:
It's kind of a strange json formatting (at least for me) but I see it is already used in thsi file. Do you know if there is reason for this?

Reply:
Yeah it's the comma-first notation. I did that initially.

When you're used to it, the relationship between delimiters and commas becomes very obvious. Also ensures line additions don't look like a diff for the previous line (because of the added comma).